Classes/Controller/FileController.php

```diff
@@ -525,12 +525,19 @@ protected function sortFiles(array $files): array
                     $isNumericSorting = true;
                     break;
                 case static::SORT_BY_NAME:
+                case 'none':
```

Comment:
position of 'none' is wrong, as "sort by name" is suppose to fall back on `default`